cis/aws/networking/aws-cis-4.2-networking-deny-public-rdp-acl-rules/aws-cis-4.2-networking-deny-public-rdp-acl-rules.sentinel

```diff
@@ -1,19 +1,19 @@
 import "tfplan/v2" as tfplan
 
 aws_security_groups = filter tfplan.resource_changes as _, resource_changes {
-	length(resource_changes.change.after.ingress else []) != 0 and
```

Comment:
@tism is it possible to replace these `length` expressions and use emptiness comparisons instead? 

I am working on reducing the usage of `length` in favour of `empty` because it is better from a readability perspective.